code/modules/mod/mod_theme.dm

```diff
@@ -1057,15 +1037,15 @@
 	//energy = 15 // 25
 
 /datum/mod_theme/prototype
-	name = "prototype"
-	desc = "A prototype modular suit powered by locomotives. While it is comfortable and has a big capacity, it remains very bulky and power-inefficient."
-	extended_desc = "This is a prototype powered exoskeleton, a design not seen in hundreds of years, the first \
-		post-void war era modular suit to ever be safely utilized by an operator. This ancient clunker is still functional, \
-		though it's missing several modern-day luxuries from updated Cybersun Industries designs. \
-		Primarily, the suit's myoelectric suit layer is entirely non-existant, and the servos do very little to \
-		help distribute the weight evenly across the wearer's body, making it slow and bulky to move in. \
-		The internal heads-up display is rendered in nearly unreadable cyan, as the visor suggests, \
-		leaving the user unable to see long distances. However, the way the helmet retracts is pretty cool."
+	name = "'Ogre' prototype"
+	desc = "A prototype modular suit developed many years ago. Incredibly heavy, power inefficient, and lacking in protection. At the very least, it's easy to modify."
+	extended_desc = "Quite possibly the first true modsuit ever created, the CSX-001 'Ogre' is a positively ancient mechanized exosuit that represents the very beginning of \
+		Cybersun Industries's foray into powered exosuits. Naturally, such an elderly design lacks virtually all of the quality of life features seen in modsuits today. \
```

Comment:
This suit comes from Theta station and was developed by Nanotrasen. smh.

Reply:
yeah but that doesn't make any sense

Reply:
> yeah but that doesn't make any sense

The lore must work within the constraints of the observed gameplay. ::P

Reply:
arghlglhlglhglhlglh